www/includes/easyparliament/templates/html/ni/index.php

```diff
@@ -0,0 +1,120 @@
+    <div class="hero">
+        <div class="row">
+        <div class="hero__mp-search">
+            <div class="hero__mp-search__wrap">
+                <h1>Do your MLAs represent you?</h1>
+                <div class="row collapse">
+                    <?php if ( count($data['regional']) > 0 ) { ?>
+                        <ul class="homepage-rep-list">
+                            <li>Your MLAs: </li>
+                        <?php foreach ( $data['regional'] as $mla ) { ?>
+                            <li class="homepage-rep-list__rep"><a href="/mla/?p=<?= $mla['person_id'] ?>"><?= $mla['first_name'] ?> <?= $mla['last_name'] ?></a></li>
```

Comment:
Still first/last name (but note other comment that using $mla['name'] here instead would be nice).